package.json

```diff
@@ -361,6 +382,8 @@
     "@octokit/rest": "^18.12.0",
     "@octokit/types": "^6.34.0",
     "@primer/components": "^31.1.0",
+    "@primer/octicons-react": "^17.0.0",
```

Comment:
might be worth matching the versions in the prototype, not a big deal though!
https://github.com/githubnext/composable-github/blob/997adebf27824e70a15c0305980baa3eedb2f520/package.json#L24